modules/json_form_widget/src/WidgetRouter.php

```diff
@@ -0,0 +1,228 @@
+<?php
+
+namespace Drupal\json_form_widget;
+
+use Drupal\Component\Uuid\Php;
+use Drupal\Core\DependencyInjection\ContainerInjectionInterface;
+use Drupal\metastore\Service;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class WidgetRouter.
+ */
+class WidgetRouter implements ContainerInjectionInterface {
```

Comment:
Sorry but i think it would be worth documenting the params and return values on at least the public functions here!